### PR TITLE
UCP/PROTO: Show protocol selection flags as part of info

### PR DESCRIPTION
**Pull request by yosefe**
## Why
Show select flags (in addition to op_attr_mask) in protocols information

### CODE REVIEW

File: src/ucp/proto/proto_debug.c [hunk 1]
```diff
@@ -317,17 +317,33 @@ void ucp_proto_select_param_str(const ucp_proto_select_param_t *select_param,
         [ucs_ilog2(UCP_OP_ATTR_FLAG_FAST_CMPL)]  = "fast-completion",
         [ucs_ilog2(UCP_OP_ATTR_FLAG_MULTI_SEND)] = "multi",
     };
+    static const uint64_t op_flag_bits = UCP_PROTO_SELECT_OP_FLAG_PPLN_FRAG |
+                                         UCP_PROTO_SELECT_OP_FLAG_AM_EAGER |
+                                         UCP_PROTO_SELECT_OP_FLAG_AM_RNDV;
+    static const char *op_flag_names[] = {
+        [ucs_ilog2(UCP_PROTO_SELECT_OP_FLAG_PPLN_FRAG)] = "frag",
+        [ucs_ilog2(UCP_PROTO_SELECT_OP_FLAG_AM_EAGER)]  = "egr",
```
Comment (shamisp) [inline src/ucp/proto/proto_debug.c:325]: `eager` is compact enough

**Review by brminich — APPROVED** [review]

File: src/ucp/proto/proto_debug.c (hunk 1 above)
Comment (yosefe) [reply src/ucp/proto/proto_debug.c:325]: all info is on same line so should be as short as possible

**Pull request merged**